nvidia-update.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-set -e
+set -eu
```

Comment:
What errors does nounset fix in this script?

Reply:
If your script is written correctly then this won't add any benefit. It is more or a safe-keeping to guarantee the script is written correctly (namely the var. you're expanding is not unset.)